src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/MigrationAssessmentLoggingHookTest.java

```diff
@@ -20,6 +20,7 @@
 
 import com.google.cloud.bigquery.dwhassessment.hooks.logger.EventLogger;
 import com.google.cloud.bigquery.dwhassessment.hooks.logger.LoggerVarsConfig;
+import com.google.cloud.bigquery.dwhassessment.hooks.test_utils.TestUtils;
```

Comment:
Package names use only lowercase letters and digits (no underscores). Consecutive words are simply concatenated together. For example, `com.example.deepspace`, not `com.example.deepSpace` or `com.example.deep_space`.

Also nit: for me `testing` seems more natural for the package name.